roles/keystone/tasks/main.yml

```diff
@@ -87,6 +87,10 @@
   notify:
     - restart keystone services
 
+- name: sync keystone database
+  command: keystone-manage db_sync
+  when: database_create.changed or force_sync|default('false')|bool
+
```

Comment:
Same deal here. A keystone restart handler is triggered by the git clone, well before we would sync the DB.